packages/wrangler/src/__tests__/vectorize/vectorize.test.ts

```diff
@@ -46,9 +46,10 @@ describe("vectorize help", () => {
 				  -v, --version                   Show version number  [boolean]
 
 				--------------------
-				📣 Vectorize is currently in open beta
+				📣 Vectorize is now Generally Available.
 				📣 Please use the '--deprecated-v1' flag to create, get, list, delete and insert vectors into legacy Vectorize indexes
-				📣 See the Vectorize docs for how to get started and known issues: https://developers.cloudflare.com/vectorize
+				📣 Creation of legacy Vectorize indexes will be blocked by December 2024. Other operations will continue to function
```

Comment:
is the intention here to no longer support the `--deprecated-v1` flag for `wrangler vectorize create` but still keep for other commands (get, list, delete, insert)?

Reply:
Correct. We will gradually deprecate V1 in three steps:
- stop new index creation in V1
- add migration tools for indexes V1 => V2
- announce V1 end of life.

Reply:
Could you add a specific warning for this case? i.e. when a user runs `create --deprecated-v1` they should get a `logger.warn("Creation of legacy Vectorize indexes will be blocked by December 2024")`

Reply:
That sounds like a good idea. Will make this change.